### PR TITLE
feat (#151): client authorization metadata

### PR DESCRIPTION
**Pull request by sliderzxc**

### CODE REVIEW

File: core/src/main/kotlin/io/timemates/backend/authorization/repositories/AuthorizationsRepository.kt [hunk 1]
```diff
@@ -32,6 +33,7 @@ interface AuthorizationsRepository {
         refreshToken: RefreshHash,
         expiresAt: UnixTime,
         creationTime: UnixTime,
+        metadata: Metadata
```
Comment (y9vad9) [inline core/src/main/kotlin/io/timemates/backend/authorization/repositories/AuthorizationsRepository.kt:36]: trailing comma

File: core/src/main/kotlin/io/timemates/backend/authorization/types/Verification.kt [hunk 2]
```diff
@@ -11,4 +12,5 @@ data class Verification(
     val attempts: Attempts,
     val time: UnixTime,
     val isConfirmed: Boolean,
+    val metadata: Metadata
```
Comment (y9vad9) [inline core/src/main/kotlin/io/timemates/backend/authorization/types/Verification.kt:15]: trailing comma

File: core/src/main/kotlin/io/timemates/backend/authorization/usecases/ConfigureNewAccountUseCase.kt [hunk 3]
```diff
@@ -56,6 +58,7 @@ class ConfigureNewAccountUseCase(
                 scopes = listOf(Scope.All),
                 expiresAt = expiresAt,
                 createdAt = currentTime,
+                metadata = metadata
```
Comment (y9vad9) [inline core/src/main/kotlin/io/timemates/backend/authorization/usecases/ConfigureNewAccountUseCase.kt:61]: trailing comma

File: data/src/main/kotlin/io/timemates/backend/data/authorization/PostgresqlVerificationsRepository.kt [hunk 4]
```diff
@@ -22,13 +23,14 @@ class PostgresqlVerificationsRepository(
...
         attempts: Attempts,
+        metadata: Metadata
     ) {
         dbVerifications.add(
             emailAddress.string,
             verificationToken.string,
             code.string,
             time.inMilliseconds,
-            attempts.int,
+            attempts.int
```
Comment (y9vad9) [inline data/src/main/kotlin/io/timemates/backend/data/authorization/PostgresqlVerificationsRepository.kt:33]: trailing comma

**Review by y9vad9 — APPROVED** [review]

**Review by y9slavabogu — APPROVED** [review]

**Pull request merged**